src/Popper.tsx

```diff
@@ -0,0 +1,42 @@
+import {
+  Popper as MaterialPopper,
+  ClickAwayListener,
+  PopperPlacementType,
+  PopperProps,
+} from "@mui/material";
+
+interface Props extends PopperProps {
+  open: boolean;
+  anchorEl: HTMLButtonElement | null;
+  placement: PopperPlacementType;
+  el: JSX.Element;
+  handleClickAway?: () => void;
+  offset: [number, number];
+}
+
+export function Popper(props: Props): JSX.Element {
+  return (
+    <ClickAwayListener onClickAway={props.handleClickAway}>
+      <MaterialPopper
+        open={props.open}
+        anchorEl={props.anchorEl}
+        style={{ display: "flex" }}
+        placement={props.placement}
+        modifiers={[
+          {
+            name: "offset",
+            options: {
+              offset: props.offset,
```

Comment:
Should we have defaults for offset and placement? Seems like the will be the same a lot of the time.

Reply:
<img width="660" alt="Screenshot 2022-03-28 at 09 08 49" src="https://user-images.githubusercontent.com/37402968/160354581-57756e22-2377-4710-a25b-72eb34f1bb7f.png">

Hmmm, I think these props are required so can't have defaults?

Reply:
unless I'm doing something wrong

Reply:
They're only required if you _make_ them required in our version of the component?

Reply:
Okay, I've just had looked at how we did it for the Iconbuttons tooltips, looks like there might be a work around